kb/deadline-exceeded-troubleshooting.md

```diff
@@ -0,0 +1,111 @@
+---
+slug: deadline-exceeded-troubleshooting
+title: DeadlineExceeded error troubleshooting
+tags:
+  - kb-article
+date: 2022-11-08T00:00:00Z
+---
+
+All requests made to the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) are [gRPC requests](https://grpc.io/docs/what-is-grpc/core-concepts/#deadlines).
+Sometimes, when these requests can't be completed, you'll see this particular error message: `Context: deadline exceeded`.
+
+`Context: deadline exceeded` is an error that occurs when requests are not completed on time.
+Network interruptions, short timeouts, server overload, and Query errors can cause this error.
+
+The following sections will discuss the nature of this error and how to troubleshoot it.
+
+#### Check your logs
+
+:::note
+
+Cloud users cannot access some of the logs needed to diagnose the source of the error.
+If you're using Temporal Cloud, consider filing a service ticket when this error occurs.
```

Comment:
```suggestion
If you're using Temporal Cloud, consider filing a [service ticket](https://support.temporal.io/) when this error occurs.
```

Reply:
My sense is there are Cloud users who don't know how to file a ticket, so linking would be helpful for them

Reply:
yes please! thanks for the link!